Regression/WarpX-tests.ini

```
@@ -1568,6 +1568,25 @@ particleTypes = beam
 analysisRoutine = Examples/analysis_default_regression.py
 tolerance = 1.e-14
 
+[Python_PlasmaAcceleration_1d]
+buildDir = .
+inputFile = Examples/Physics_applications/plasma_acceleration/PICMI_inputs_plasma_acceleration_1d.py
+runtime_params =
+customRunCmd = python PICMI_inputs_plasma_acceleration_1d.py
+dim = 1
+addToCompileString = USE_PYTHON_MAIN=TRUE PYINSTALLOPTIONS="--user --prefix=" USE_PSATD=FALSE
```

```suggestion
addToCompileString = USE_PYTHON_MAIN=TRUE PYINSTALLOPTIONS="--user --prefix="
```

discussed: this is using PICMI w/o 1D classes yet